api/Document.json

```diff
@@ -8985,6 +9032,53 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "line_param_accepts_TrustedHTML": {
+          "__compat": {
+            "description": "`text` parameter accepts `TrustedHTML` type",
```

Comment:
param is called `line` in the page.

Reply:
As per https://github.com/mdn/browser-compat-data/pull/25839#discussion_r1945606107 I'm going to change this to markup in the docs too (not line/text) as here. Will follow same pattern as the above link.